src/azure-cli/azure/cli/command_modules/vm/tests/hybrid_2019_03_01/test_vm_image.py

```diff
@@ -27,7 +27,7 @@ def _get_test_cmd():
 
 
 class TestVMImage(unittest.TestCase):
-    @mock.patch('azure.cli.command_modules.vm.custom.urlopen', autospec=True)
+    @mock.patch('azure.cli.command_modules.vm.custom.requests', autospec=True)
     def test_read_images_from_alias_doc(self, mock_urlopen):
```

Comment:
Good question!

Previously, the purpose of this mock was to mock the querying results of `https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/quickstart-templates/aliases.json` when the method `all_images = load_images_from_aliases_doc(cmd.cli_ctx, publisher_name, offer, sku)` was called.
I speculate that this is to prevent the test from blocking when the remote file cannot be accessed, so the local cache file `aliases.json` is used to mock the access results of the remote file.


However, since there was a new PR https://github.com/Azure/azure-cli/pull/10852 to add the logic of reading local json dict when remote files cannot be accessed to the method `load_images_from_aliases_doc()`
```Python
    except requests.exceptions.ConnectionError:
        logger.warning("Failed to retrieve image alias doc '%s'. Error: 'ConnectionError'. Use local copy instead.",
                       target_url)
        dic = json.loads(alias_json)
```
[code link](https://github.com/Azure/azure-cli/pull/10852/files#diff-d7da56dc08a2c634ed0f2baf9094f24d90de84164ba83d2334863b77280379e8L82-R97)

Since the logic of thoroughly querying `alias_json` in method `load_images_from_aliases_doc()` has replaced the function of mock logic, so even if the mock logic is wrong, they will not affect the process of `load_images_from_aliases_doc`. Therefore, these mock logics can be deleted directly.